src/components/Card/Card.js

```diff
@@ -4,6 +4,27 @@ import { getChannelInfo } from "../../channelProvider";
 import classNames from "classnames";
 import countries from "svg-country-flags/countries.json";
 
+
+
+
+// ========Track Analytics Events on Page =======
+function handleAnalytic(channelName) {
+  console.log("I AM FIRING THIS FUNCTION")
```

Comment:
Don't forget to delete it. We're not need `console` on production.

Reply:
Alright, I will clear this up and update the PR